### PR TITLE
Use packageurl version 0.9.3 and Add nginx importer

### PR DESCRIPTION
**Pull request by sbs2001**
Signed-off-by: Shivam Sandbhor <shivam.sandbhor@gmail.com>

### CODE REVIEW

**Comment by sbs2001** [comment]
Fixes https://github.com/nexB/vulnerablecode/issues/96

File: vulnerabilities/importers/nginx.py [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+#  VulnerableCode should be considered or used as legal advice. Consult an
+#  for any legal advice.
+#  VulnerableCode is a free software  from nexB Inc. and others.
+#  Visit https://github.com/nexB/vulnerablecode/ for support and download.
+
+import asyncio
+import dataclasses
+import json
+
+import requests
```
Comment (pombredanne) [inline vulnerabilities/importers/nginx.py:27]: Nit: we should sort imports. But we should also switch to using black and isort, so we can do it all at once!

File: vulnerabilities/importers/nginx.py [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+                    summary=summary,
+                    impacted_package_urls=vulnerable_packages,
+                    resolved_package_urls=fixed_packages,
+                )
+            )
+
+        return advisories
+
+    def extract_fixed_pkgs(self, vuln_info):
+        vuln_status, version_info = vuln_info.split(": ")
```
Comment (pombredanne) [inline vulnerabilities/importers/nginx.py:133]: Using a `.partition()` is more robust as it always does a single split and always return three values making the tuple unpacking safer and more reliable. I tend to prefer it followed by a `.strip()` call.

**Review by pombredanne — APPROVED** [review]
I am merging this leave aside the few nitpickings... we should definitely add tests though!
Thanks!

**Pull request merged**